src/mono/nuget/Microsoft.NET.Runtime.MonoAOTCompiler.Workload/WorkloadManifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": 1,
+    "workloads": {
+      "microsoft-net-runtime-monoaotcompiler-workload": {
```

Comment:
Mikayla recommended making this more generic. Basically have 1 workload manifest for all Mono packs.

Reply:
If the Android, iOS, etc. workloads use [extends](https://github.com/mhutch/designs/blob/b82449a228c0addb95b5a4995bb838749ea6f8cc/accepted/2020/workloads/workload-manifest.md#workload-definitions) on this pack. I guess we just need make sure that it's OK that everything listed in this manifest will be installed for those workloads.

Reply:
@mhutch thoughts on that?